spec/human_player_spec.rb

```diff
@@ -16,31 +20,25 @@ def output(message)
     end
 
     def input
-      @inputs.shift
+      @input.shift
     end
   end
 
-  class Player::FakeBoard
-    attr_reader :selection
-
-    def initialize
-      @selection = 1
-    end
-
-    def is_available?(position)
-      (1..9).cover?(position) unless position == selection
-    end
+  class BoardWithNoMethods end
 
-    def next_token
-      "X"
+  class ValidatorWithoutOneAvailable
+    def validate(selection, board)
+      return OpenStruct.new(status: :invalid_entry, position: nil) unless selection.between?(1, 9)
+      return OpenStruct.new(status: :space_taken, position: nil) if selection == 1
+      OpenStruct.new(position: selection)
```

Comment:
This looks a little suspicious given that it nearly identical to the `SelectionValidator`. Does it need to be?

If so,  it might indicate that that the coupling is high between `HumanPlayer` and `SelectionValidator`. Maybe something to keep an eye on.

Reply:
I tried to simplify but was having difficulty...